### PR TITLE
[PLAT-254] Remove is_creator usage

### PR DESCRIPTION
**Pull request by raymondjacobson**
### Description

Remove is_creator usage in client.
https://github.com/AudiusProject/audius-protocol/pull/3475

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

Local stack, local client, linked sdk
- make account
- view profile without tracks
- upload track (no 1% stickiness)
- view track
- view profile
- upload second track
- update profile
- sign out / sign in
- view artist dashboard
- view cnode selection modal in settings

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-plat-254

File: packages/web/src/pages/upload-page/components/FinishPage.js [hunk 1]
```diff
@@ -153,12 +151,8 @@ class FinishPage extends Component {
           </div>
         )
 
-        // If we're waiting around to upgrade to become a creator,
-        // fake out the user by setting everything to 1% so it doesn't
-        // look totally stalled.
-        const uploadPercent = isCreator
-          ? (uploadProgress[i].loaded / uploadProgress[i].total) * 100
-          : 1
+        const uploadPercent =
```
Comment (dylanjeffers) [inline packages/web/src/pages/upload-page/components/FinishPage.js:154]: nice change

File: packages/web/src/store/account/sagas.js [hunk 2]
```diff
@@ -430,22 +429,6 @@ function* fetchSavedPlaylistsAsync() {
   })
 }
 
-function* setAccountCreator(action) {
```
Comment (dylanjeffers) [inline packages/web/src/store/account/sagas.js:433]: nice clean up

File: package-lock.json [hunk 3]
```diff
@@ -2010,7 +2010,7 @@
     "add-stream": {
       "version": "1.0.0",
       "resolved": "https://registry.npmjs.org/add-stream/-/add-stream-1.0.0.tgz",
-      "integrity": "sha512-qQLMr+8o0WC4FZGQTcJiKBVC59JylcPSrTtk6usvmIDFUOCKegapy1VHQwRbFMOFyb/inzUVqHs+eMYKDM1YeQ==",
```
Comment (dylanjeffers) [inline package-lock.json:2013]: we need lock chages? wonder what's happening here cause ive seen this a few times lately

Comment (raymondjacobson) [reply package-lock.json:2013]: really not sure. ill undo them here

Comment (raymondjacobson) [reply package-lock.json:2013]: npm version maybe?

**Comment by raymondjacobson** [comment]
> nice cleanup. so from what i understand, we are removing the concept of is_creator, and just checking if they have tracks? just curious, as the user is uploading their first track, when is the point where the app views them as an artist? once we get to upload finished screen?

Yep that's right. There's no logical separation between user & creator like there once was. is_creator came from a world where non-artists had no tracks and they were stored on a different node type: the user metadata node. that concept is over with.

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-plat-254

**Pull request merged**